Formula/secman.rb

```diff
@@ -0,0 +1,29 @@
+class Secman < Formula
+  desc "👊 Human-friendly and amazing secrets manager"
+  homepage "https://secman.dev/docs/cli"
+  url "https://cli-files.secman.dev/secman-v6.0.71/secman-v6.0.71.tar.gz"
```

Comment:
Why aren't we using the GitHub repo here?

Reply:
it's the same file

Reply:
If it's the same we should use the GitHub repo.